enterprise_subsidy/apps/api/v1/tests/test_views.py

```diff
@@ -141,10 +141,6 @@ class SubsidyViewSetTests(APITestBase):
     get_list_url = partial(reverse, "api:v1:subsidy-list")
     get_can_redeem_url = partial(reverse, "api:v1:subsidy-can-redeem")
 
-    def setUp(self):
-        super().setUp()
-        self.subsidy_1.catalog_client = mock.MagicMock()
-
```

Comment:
This wasn't doing anything anyway because the subsidy model instance from the unit test's perspective is a different instance than from the view's perspective, despite them both referring to the same DB object.